serving/docs/lmi/tutorials/trtllm_aot_tutorial.md

```diff
@@ -1,3 +1,102 @@
-# TensorRT LLM ahead of time compilation of models
+# TensorRT-LLM ahead-of-time compilation of models tutorial
 
-This doc helps you to convert your HuggingFace model to Tensorrt-LLM LMI model format to load and run inference with Tensorrt-LLM.
\ No newline at end of file
+## Overview
+
+With LMI TensorRT-LLM container, you can run on the fly compilation of Falcon and Llama architecture models and then load the model and run inference. However, for larger models like Falcon 40B and Llama-2 70B models, compilation of models takes approximately 7 minutes and more. So, we recommend users to perform ahead-of-time compilation for these larger models and other model architectures to avoid this compilation overhead.
+
+The goal of this document is for the user to be able to:
+
+1. Convert the HuggingFace model to TensorRT-LLM model format.
+2. Upload the model to S3 so you could use it for inference. 
+
+## Step by step tutorial
+
+In this tutorial, we will be converting the Llama2-70b model to TensorRT-LLM model format on p4d.24xlarge.
+
+### Step 1: Choose your instance
+
+To do the model compilation you need to use **** the instance that has **the same GPU architecture** **** that will be used for your inference deployment **** (e.g model compiled on a g5 instance can be deployed only on g5 instances).
```

Comment:
Let's remove the extra asterisks in this sentence.

Reply:
Done